code/modules/admin/verbs/randomverbs.dm

```diff
@@ -110,5 +110,16 @@ var/global/explosive_antigrief_on = TRUE
 	set name = "Toggle Explosive Antigrief"
 	set category = "Admin.Game"
 
-	explosive_antigrief_on = !explosive_antigrief_on
-	message_staff(FONT_SIZE_LARGE("[key_name_admin(usr)] has [explosive_antigrief_on ? "enabled" : "disabled"] explosive antigrief"))
+	switch(CONFIG_GET(number/explosive_antigrief))
```

Comment:
this should make a TGUI input list / alert for the admin to pick rather than having to switch through all of them, also there should be a way for admins to check what setting the antigrief is currently on